common/src/attestations/signers.rs

```diff
@@ -21,7 +19,7 @@ pub struct AttestationSigners {
 }
 
 #[derive(Debug)]
-struct AttestationSignersInner {
+pub(crate) struct AttestationSignersInner {
```

Comment:
We don't need to make this public as of now.
```suggestion
struct AttestationSignersInner {
```

Reply:
@aasseman You were right. Fixed!